### PR TITLE
Setup associations

### PR DESCRIPTION
**Pull request by initseis**
## Description
In this branch I:

- Added a Book class
- Added a Classroom class
- Added a Rental Class
- Updated the Person Class
- Updated the Student Class

## How Has This Been Tested?
- I added a test.rb file to do the tests of this part.

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```
Comment (Emmyn5600) [inline book.rb]: - [ ] Please add a method for adding rentals should be present

File: person.rb [hunk 2]
```diff
@@ -2,14 +2,15 @@
...
+  attr_reader :id, :rentals
 
   def initialize(age, parent_permission = true, name = 'Unknown')
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
     @corrector = Corrector.new
+    @rentals = []
   end
```
Comment (Emmyn5600) [inline person.rb:15]: - [ ] Should add a method for adding a rental

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

### Hi, @sevinchek 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:
   
 - The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

   ###  Changes required :recycle: 
   
    _Check the comment under the review_

    Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by DeliceLydia — APPROVED** [review]
## Hi @sevinchek, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by initseis** [comment]
Thanks, @DeliceLydia @Emmyn5600.

**Pull request merged**